core/Network/TLS/Extra/Cipher.hs

```diff
@@ -652,6 +655,28 @@ cipher_ECDHE_RSA_AES256GCM_SHA384 = Cipher
     , cipherMinVer       = Just TLS12 -- RFC 5289
     }
 
+cipher_AES128GCM_SHA256 :: Cipher
+cipher_AES128GCM_SHA256 = Cipher
+    { cipherID           = 0x1301
+    , cipherName         = "AES128GCM-SHA256"
+    , cipherBulk         = bulk_aes128gcm
+    , cipherHash         = SHA256
+    , cipherPRFHash      = Nothing
+    , cipherKeyExchange  = CipherKeyExchange_TLS13
+    , cipherMinVer       = Just TLS13ID16
```

Comment:
I am concerned about introducing draft-versions into stable code.  Is this intended for a non-prod branch of Network.TLS? The stable version of Network.TLS should have no support for TLS13ID16 or similar.  There should only be TLS13.

Since the cipher code point has already been assigned by IANA, I see no reason to use TLS13ID16 here.  Just go with TLS13 already, and by the time this code appears in a stable release it should match the final TLS 1.3 definition.